langchain/agents/chat/output_parser.py

```diff
@@ -17,13 +17,13 @@ def parse(self, text: str) -> Union[AgentAction, AgentFinish]:
         try:
             action = text.split("```")[1]
             response = json.loads(action.strip())
-            includes_action = "action" in response and "action_input" in response
+            includes_action = "action" in response
             if includes_answer and includes_action:
                 raise OutputParserException(
                     "Parsing LLM output produced a final answer "
                     f"and a parse-able action: {text}"
                 )
-            return AgentAction(response["action"], response["action_input"], text)
+            return AgentAction(response["action"], response.get("action_input", ""), text)
```

Comment:
response.get("action_input", {})

Reply:
If the tool indeed doesn't accept any args

Reply:
fixed